templates/common/baremetal/units/nodeip-configuration.service.yaml

```diff
@@ -13,14 +13,22 @@ contents: |
   [Service]
   # Need oneshot to delay kubelet
   Type=oneshot
-  ExecStart=/usr/bin/podman run --rm \
+  # Would prefer to do Restart=on-failure instead of this bash retry loop, but
```

Comment:
`Restart=on-failure` has been in systemd forever, what feature do we need from systemd v244?

https://github.com/systemd/systemd/blob/master/NEWS

Reply:
Restart=on-failure was not available for `oneshot` until v244 IIRC, only for the other kinds of [Service]